packages/ariakit/src/combobox/combobox-item.tsx

```diff
@@ -73,14 +82,16 @@ export const useComboboxItem = createHook<ComboboxItemOptions>(
 
     const onClickProp = useEventCallback(props.onClick);
 
+    const hideOnClickProp = useBooleanEventCallback(hideOnClick);
```

Comment:
Nit:
```suggestion
    const onClickProp = useEventCallback(props.onClick);
    const hideOnClickProp = useBooleanEventCallback(hideOnClick);
```

Reply:
Got it. To get familiar with the general style of the project, this is because single-line statements are usually put together, with no line breaks in-between, right?